<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraClass</class>
 <widget class="QWidget" name="CameraClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera</string>
  </property>
  <widget class="QPushButton" name="camera1_btn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>330</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="camera2_btn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QWidget" name="display_widget" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>391</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="capture_btn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
